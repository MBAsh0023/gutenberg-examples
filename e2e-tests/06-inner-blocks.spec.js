/**
 * WordPress dependencies
 */
import {
	getEditedPostContent,
	insertBlock,
	createNewPost,
} from '@wordpress/e2e-test-utils';

/**
 * Internal dependencies
 */
import json from '../06-inner-blocks/block.json';
const { title, name } = json;

it( `Example: ${ title } block should be available`, async () => {
	await createNewPost();
	await insertBlock( title );

	// Check if block was inserted
	expect( await page.$( `[data-type="${ name }"]` ) ).not.toBeNull();

	expect( await getEditedPostContent() ).toMatchSnapshot();
} );
