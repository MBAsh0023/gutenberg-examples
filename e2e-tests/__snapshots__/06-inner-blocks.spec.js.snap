// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example: Example: Inner Blocks block should be available 1`] = `
"<!-- wp:gutenberg-examples/example-06 -->
<div class=\\"wp-block-gutenberg-examples-example-06\\"></div>
<!-- /wp:gutenberg-examples/example-06 -->"
`;
